python/activator/middleware_interface.py

```diff
@@ -128,7 +118,7 @@ def _init_local_butler(self, butler: Butler):
         # self.instrument.register(butler.registry)
 
         # Refresh butler after configuring it, to ensure all required
-        # dimensions are available.
+        # collections are available.
```

Comment:
As it stands, init_butler has nothing to do, since we have to put off all the work for prep_butler right now. We can leave it around while we wait for middleware fixes, etc. that will let us do more things here, but I've added a note to the method docs about whether it's worth keeping the method.

Reply:
My above comment is now out of date, since Jim gave us a fix for the `instrument.register` problem.